Modules/Sources/Support/UI/Application Logs/ActivityLogListView.swift

```diff
@@ -0,0 +1,173 @@
+import SwiftUI
+
+/// A view that displays a list of application log files in reverse chronological order.
+public struct ActivityLogListView: View {
+
+    enum ViewState {
+        case loading
+        case loaded([ApplicationLog], DeletionState)
+        case error(Error)
+    }
+
+    enum DeletionState {
+        case none
+        case confirm
+        case deleting(Task<Void, Never>)
+        case deletionError(Error)
+    }
+
+    @EnvironmentObject
+    private var dataProvider: SupportDataProvider
+
+    @State
+    var state: ViewState = .loading
+
+    @State
+    var isConfirmingDeletion: Bool = false
+
+    public init() {}
+
+    public var body: some View {
+        Group {
+            switch self.state {
+            case .loading:
+                loadingView
+            case .loaded(let logFiles, let deletionState):
+                listView(logFiles: logFiles, deletionState: deletionState)
+            case .error(let error):
+                ErrorView(
+                    title: "Error loading logs",
+                    message: error.localizedDescription
+                )
+            }
+        }
+        .navigationTitle("Activity Logs")
```

Comment:
Should this be "Application Logs", which is what the list row says?

Reply:
Yeah, good call – done in 10bc23534403c29aa14f07f43af428779efb52d8